atd-vze/src/views/Crashes/Crashes.js

```diff
@@ -1,72 +1,166 @@
 import React from "react";
 import { withApollo } from "react-apollo";
-import TableWithFilters from "../../Components/TableWithFilters";
-import { crashDataMap } from "./crashDataMap";
 
-// Key for data response array from DB
-const dataKey = "atd_txdot_crashes";
+import GridTable from "../../Components/GridTable";
 
-const GET_CRASHES = `
-  {
-    atd_txdot_crashes(
-      offset: 0
-      where: { crash_fatal_fl: { _eq: "Y" } }
-    ) {
-      crash_id
-      death_cnt
-      tot_injry_cnt
-      crash_fatal_fl
-      rpt_street_pfx
-      rpt_street_sfx
-      rpt_street_name
-      crash_date
-    }
-  }
-`;
+import gqlAbstract from "../../queries/gqlAbstract";
 
-const FILTER_CRASHES = `
-  {
-    atd_txdot_crashes(
-      OFFSET
-      LIMIT
-      ORDER_BY
-      SEARCH
-    ) {
-      crash_id
-      death_cnt
-      tot_injry_cnt
-      crash_fatal_fl
-      rpt_street_pfx
-      rpt_street_sfx
-      rpt_street_name
-      crash_date
-    }
-  }
-`;
+// Our initial query configuration
+let queryConf = {
+  table: "atd_txdot_crashes",
+  single_item: "crashes",
+  columns: {
+    crash_id: {
+      primary_key: true,
+      searchable: true,
+      sortable: true,
+      label_search: "Search by Crash ID",
+      label_table: "Crash ID",
+      type: "Int",
+    },
+    case_id: {
```

Comment:
I don't think we need case_id as a column. Do you remember that request? I know you had it in the QA grid table but do you remember if there was a specific request to add it?